pkg/sql/privilege/privilege.go

```diff
@@ -31,6 +31,7 @@ type Kind uint32
 const (
 	_ Kind = iota
 	ALL
+	CONNECT
```

Comment:
Careful - this would need to come at the end, because this is actually a bitfield that's used to encode and decode values on disk! (side note we should probably move it to a protobuf to avoid potential mistakes like this)

Reply:
Good catch, thanks! I'll leave a comment for now saying to add to the end.

Reply:
@RichardJCai I'm going to give you https://github.com/cockroachdb/cockroach/issues/58675 given this has come up a few times recently.